src/ontology/uberon-edit.obo

```diff
@@ -184,6 +184,7 @@ property_value: doap-SVNRepository "https://obo.svn.sourceforge.net/svnroot/obo/
 property_value: foaf-homepage "http://uberon.org" xsd:anyURI
 property_value: has_ontology_root_term UBERON:0000104
 property_value: has_ontology_root_term UBERON:0001062
+owl-axioms: Prefix(owl:=<http://www.w3.org/2002/07/owl#>)\nPrefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)\nPrefix(xml:=<http://www.w3.org/XML/1998/namespace>)\nPrefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)\nPrefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)\n\n\nOntology(\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0002397>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_0005813>))\nDeclaration(Class(<http://purl.obolibrary.org/obo/UBERON_8500005>))\nDeclaration(ObjectProperty(<http://purl.obolibrary.org/obo/BFO_0000050>))\n\n############################\n#   Classes\n############################\n\n# Class: <http://purl.obolibrary.org/obo/UBERON_8500005> (<http://purl.obolibrary.org/obo/UBERON_8500005>)\n\nSubClassOf(<http://purl.obolibrary.org/obo/UBERON_8500005> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/UBERON_0005813> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0002397>)))\n\n\n)
```

Comment:
This is the same problem as in #3516.

You are creating the following axiom:

> 'facial tubercle' **SubClassOf:** 'tubercle' **and** ('part of' **some** 'maxilla')

The OBO format does not allow to represent **SubClassOf** axioms where the superclass is an ObjectIntersectionOf, so the axiom ends up in this `owl-axioms` tag.

According to the new term request in #2951, what was requested is an equivalence axiom instead:

> 'facial tubercle' **EquivalentTo:** 'tubercle' **and** ('part of' **some** 'maxilla')